CONTRIBUTING.md

```diff
@@ -62,7 +62,7 @@ issues and/or Redmine tickets (if applicable).
     4. For SQL patches that are _optional_ (e.g. those that perform some 
             cleanup), place them in `SQL/Cleanup_patches/`.
     5. The files for our test database (Raisinbread) must be updated if they were modified.
-        Detailed instructions for this step can be found [here](https://github.com/aces/Loris/tree/minor/raisinbread#exporting-rb).
+        Detailed instructions for this step can be found in the Raisin Bread [README.md](./raisinbread#exporting-rb).
```

Comment:
```suggestion
        Detailed instructions for this step can be found in the Raisin Bread [README.md](./raisinbread/README.md#exporting-rb).
```

I meant it should be explicitly in the URL in my last review.. but I think the wording of the text part that you've changed it also an improvement

Reply:
Oh sorry, I misunderstood your comment! Fixed!